source/pages/Settings/PrivateKey.tsx

```diff
@@ -91,14 +95,35 @@ const PrivateKeyView = () => {
         <CopyCard
           className="my-3"
           onClick={
-            valid ? () => copyText(String(activeAccount?.xprv)) : undefined
+            valid
+              ? () =>
+                  copyText(
+                    String(
+                      isSyscoinChain
```

Comment:
also we have both sys & evm private keys encrypted, so no need to check the network to decrypt